granary/tests/test_bluesky.py

```diff
@@ -335,6 +335,29 @@
   'alt': 'my alt text',
 }
 
+POST_AS_AUDIO = copy.deepcopy(POST_AS)
+POST_AS_AUDIO['object']['attachments'] = [{
+  'objectType': 'audio',
+  'displayName': 'my alt text',
+  'stream': {
+    'url': NEW_BLOB_URL,
```

Comment:
Minor nit: could you use a URL that's not on bsky.app here, to avoid any confusion? And then also update the embed below to match.